spring-batch-docs/asciidoc/domain.adoc

```diff
@@ -765,4 +767,3 @@ As long as the batch namespace has been declared, any of its
 elements can be used. More information on configuring a
 Job can be found in <<job.adoc#configureJob,Configuring and Running a Job>>. More information on configuring a `Step` can be
 found in <<step.adoc#configureStep,Configuring a Step>>.
-
```

Comment:
We leave the blank line at the end because it makes diffs nicer with git.

Reply:
Interesting. My file has a blank line at the end. I wonder why it didn't make it into Github.